libnopegl/src/backends/gl/gpu_ctx_gl.c

```diff
@@ -970,32 +970,27 @@ static int gl_get_preferred_depth_stencil_format(struct gpu_ctx *s)
 
 static void gl_set_pipeline(struct gpu_ctx *s, struct pipeline *pipeline)
 {
-    struct gpu_ctx_gl *s_priv = (struct gpu_ctx_gl *)s;
-    s_priv->current_pipeline = pipeline;
 }
```

Comment:
Does it make sense to keep this callback?

Reply:
It does, later on, it will directly bind the pipeline (ie: glUseProgram(), and vkCmdBindPipeline()).